lib/public/views/Home/Overview/HomePage.js

```diff
@@ -14,33 +14,46 @@
 import { h } from '/js/src/index.js';
 import { logsActiveColumns } from '../../Logs/ActiveColumns/logsActiveColumns.js';
 import { runsActiveColumns } from '../../Runs/ActiveColumns/runsActiveColumns.js';
+import { lhcFillsActiveColumns } from '../../LhcFills/ActiveColumns/lhcFillsActiveColumns.js';
 import { table } from '../../../components/common/table/table.js';
 import { estimateDisplayableRowsCount } from '../../../utilities/estimateDisplayableRowsCount.js';
 
 const TABLEROW_HEIGHT = 46;
 // Estimate of the navbar and pagination elements height total; Needs to be updated in case of changes;
 const PAGE_USED_HEIGHT = 155;
+const MAX_ROWS = 7;
 
 /**
  * Home Page component
  * @param {Object} model global model
  * @return {Component} Return the component of the home page
  */
-export const HomePage = ({ home: { logsOverviewModel, runsOverviewModel } }) => {
-    const rowCount = estimateDisplayableRowsCount(TABLEROW_HEIGHT, PAGE_USED_HEIGHT);
-    logsOverviewModel.pagination.provideDefaultItemsPerPage(rowCount);
-    runsOverviewModel.pagination.provideDefaultItemsPerPage(rowCount);
+export const HomePage = ({ home: { logsOverviewModel, runsOverviewModel, lhcFillsOverviewModel } }) => {
+    const rowCount = Math.round(estimateDisplayableRowsCount(TABLEROW_HEIGHT, PAGE_USED_HEIGHT) / 2) - 2;
```

Comment:
Why `/2)- 2`? You should update (maybe create another constant if needed) `PAGE_USED_HEIGHT`